week-1-practise/level-2/3-currency-conversion.js

```diff
@@ -5,7 +5,10 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(price) {
+  // conversion price
+  return price * 1.4;
```

Comment:
Same thing about the comments, but great solution!

Reply:
Can I say "format currency"